docs/variants/dell_optiplex/building-manual.md

```diff
@@ -1,53 +1,122 @@
 # Dell OptiPlex 7010/9010 Dasharo - building manual
 
-## Building coreboot
+## Release build
 
 To build coreboot image, follow the steps below:
```

Comment:
`coreboot image` or `Dasharo firmware image`, or something like that?

Reply:
@macpijan fixed https://github.com/Dasharo/docs/pull/139/commits/dd6bc1602856faaa256acb4dff5bfea06368c591